launch_param_builder/utils.py

```diff
@@ -62,17 +77,23 @@ def load_file(file_path: Path):
     try:
         with open(file_path, "r") as file:
             return file.read()
-    except EnvironmentError:  # parent of IOError, OSError *and* WindowsError where available
+    except OSError:  # parent of IOError, OSError *and* WindowsError where available
         return None
 
 
 def load_yaml(file_path: Path):
     raise_if_file_not_found(file_path)
 
+    try:
+        yaml.SafeLoader.add_constructor("!radians", construct_angle_radians)
+        yaml.SafeLoader.add_constructor("!degrees", construct_angle_degrees)
+    except Exception:
+        raise Exception("yaml support not available; install python-yaml")
+
     try:
         with open(file_path, "r") as file:
-            return yaml.load(file, Loader=yaml.FullLoader)
-    except EnvironmentError:  # parent of IOError, OSError *and* WindowsError where available
+            return yaml.safe_load(file)
```

Comment:
Based on [this resource](https://matthewpburruss.com/post/yaml/) and some limitation I found when integrating this in Studio, I have a suggestion.

First, make a function like this which we can use standalone:

```
def get_yaml_loader():
    """Gets a YAML loader that additionally supports custom constructors."""
    loader = yaml.SafeLoader
    loader.add_constructor("!radians", construct_angle_radians)
    loader.add_constructor("!degrees", construct_angle_degrees)
    return loader
  ```

Then, in your implementation, you can do:

```
def load_yaml(file_path: Loader, loader=get_yaml_loader()):
    ...
    return yaml.load(file, Loader=loader)
````